mopidy/mpd/protocol/stored_playlists.py

```diff
@@ -157,7 +166,15 @@ def playlistclear(context, name):
 
         Clears the playlist ``NAME.m3u``.
     """
-    raise exceptions.MpdNotImplemented  # TODO
+    uri = context.lookup_playlist_uri_from_name(name)
+    playlist = uri is not None and context.core.playlists.lookup(uri).get()
+    if not playlist:
```

Comment:
If we wait with adding some way of probing what the playlist supports just picking one is probably the best we can do. Though perhaps we could just add a `mpd/fallback_playlist_provider` setting to at least make it explicit?